libs/src/services/dataContracts/index.js

```diff
@@ -137,7 +137,7 @@ class AudiusContracts {
     // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#Computed_property_names
     this.contracts = this.contracts || { [this.registryAddress]: 'registry' }
     this.contractAddresses = this.contractAddresses || { 'registry': this.registryAddress }
-    if (!this.contractAddresses[contractName]) {
+    if (!this.contractAddresses[contractName] || Utils.isZeroAddress(this.contractAddresses[contractName])) {
```

Comment:
this is the bug that took me several hours lol

also will be moved to separate PR libs only

Reply:
fwiw i repro'd this on client by making a user prior to contract deploy